### ISSUE TITLE
Initializing tablesorter from a JavaScript array or JSON data

### ISSUE THREAD

**Issue by dandv** [Mottie/tablesorter#500]
I'm working on a [Meteor](http://meteor.com) application, which makes data on the wire _very_ nicely available, without having to send HTML. All data resides on the client.

Is it possible to initialize the tablesorter with data from an array object, or maybe from a JSON string? I've scanned the [docs](http://mottie.github.io/tablesorter/docs/) and issue #31 and if such an option exists, it would be great to emphasize it a bit in the docs.

Basically, I'm looking for [Dynatable's existing JSON functionality](http://www.dynatable.com/#existing-json).
